addons/website/views/snippets/snippets.xml

```diff
@@ -524,8 +524,9 @@
         data-exclude=".s_col_no_resize.row > div, .s_col_no_resize"/>
 
     <t t-set="so_snippet_addition_selector" t-translation="off">section, .parallax, .s_popup</t>
+    <t t-set="so_snippet_addition_only_drop_in_selector" t-translation="off">.s_popup</t>
     <div id="so_snippet_addition"
-        t-att-data-selector="so_snippet_addition_selector"
+        t-attf-data-selector="#{so_snippet_addition_selector}, #{so_snippet_addition_only_drop_in_selector}"
```

Comment:
I don't understand this change, what is this about? You repeat the popup selector twice, in the same attribute :thinking: I don't think it does anything (note it conflict with a recent fix too, I just removed it in my conflict resolution).

Reply:
That's a mistake, this change is useless... Sorry for that.